test/inductor/test_fused_attention.py

```diff
@@ -165,6 +165,42 @@ def sfdp_pattern_6(query, key, value):
 
         self._check_common(sfdp_pattern_6, contains=False)
 
+    def test_pattern_fails_with_tensor_factor(self):
+        # https://github.com/pytorch/pytorch/issues/99124
```

Comment:
This issue is reported on CPU side, but I checked it is also happening on GPU side, so add this case here.